pymtl3/passes/WaveGenPass.py

```diff
@@ -0,0 +1,189 @@
+from __future__ import absolute_import, division, print_function
+
+import time
+from collections import defaultdict
+from copy import deepcopy
+
+import py
+import sys
+from pymtl3.dsl import Const
+from pymtl3.passes.BasePass import BasePass, PassMetadata
+
+from .errors import PassOrderError
+
+
+
+class WaveGenPass( BasePass ):

```

Comment:
What is the difference between `WaveGenPass` and `SignalCollectPass`? Only the latter pass is included in the pass group so `WaveGenPass` seems to be unused?

Reply:
Yeah WaveGenPass should not be included.  I guess I should cancel some push changes?

Reply:
It's OK if you decide some of the code should not be included -- just remove them in a new commit!